aws-lambda-scorer/lambda-template/src/main/java/ai/h2o/mojos/deploy/aws/lambda/MojoScorer.java

```diff
@@ -3,10 +3,7 @@
 
 import ai.h2o.mojos.deploy.common.rest.model.ScoreRequest;
 import ai.h2o.mojos.deploy.common.rest.model.ScoreResponse;
-import ai.h2o.mojos.deploy.common.transform.MojoFrameToResponseConverter;
-import ai.h2o.mojos.deploy.common.transform.RequestChecker;
-import ai.h2o.mojos.deploy.common.transform.RequestToMojoFrameConverter;
-import ai.h2o.mojos.deploy.common.transform.ScoreRequestFormatException;
+import ai.h2o.mojos.deploy.common.transform.*;
```

Comment:
![no](https://media.giphy.com/media/12XMGIWtrHBl5e/giphy.gif)

Don't use wildcard imports please. 

https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Let me blame this on my IntelliJ settings ;o). Fixed.